error-prone-contrib/src/test/java/tech/picnic/errorprone/bugpatterns/DirectReturnTest.java

```diff
@@ -223,4 +225,26 @@ void replacement() {
             "}")
         .doTest(TestMode.TEXT_MATCH);
   }
+
+  @Test
+  void lombokDataReplacement() {
+    BugCheckerRefactoringTestHelper.newInstance(DirectReturn.class, getClass())
+            .addInputLines(
+                    "AnyClass.java",
+                    "import lombok.Data;",
+                    "",
+                    "@Data",
+                    "public class AnyClass {",
+                      "private String anything;",
+                    "}")
+            .addOutputLines(
+                    "AnyClass.java",
+                    "import lombok.Data;",
+                    "",
+                    "@Data",
+                    "public class AnyClass {",
+                      "private String anything;",
+                    "}")
```

Comment:
For reference, here is the actual output 😢 
```suggestion
    import lombok.Data;
    
    return result * PRIME + ($anything == null ? 43 : $anything.hashCode());
    public class AnyClass {
    private String anything;
    }
```